packages/graphql/package.json

```diff
@@ -47,5 +49,32 @@
     "src",
     "schemas"
   ],
-  "types": "src/index.ts"
+  "types": "src/index.ts",
+  "nx": {
+    "targets": {
+      "nexus-build": {
+        "outputs": [
+          "{projectRoot}/src/gen/nxs.gen.ts",
+          "{projectRoot}/schemas/schema.graphql"
+        ]
+      },
+      "build": {
+        "dependsOn": [
+          "nexus-build"
+        ],
+        "outputs": [
+          "{workspaceRoot}/packages/frontend-shared/cypress/support/generated/test-graphql-types.gen.ts",
```

Comment:
It's worth noting that we should eventually move away from one lib having outputs in another. It's fine for now but we should consider refactoring this at some point